Wikipedia/Categories/UIImage+WMFStyle.m

```diff
@@ -14,4 +14,15 @@ + (UIImage*)wmf_placeholderImage {
     return img;
 }
 
++ (instancetype)wmf_imageFromColor:(UIColor*)color {
+    CGRect rect = CGRectMake(0, 0, 1, 1);
```

Comment:
> i'm guessing backgroundColor didn't cut it?

Produces rounded corners.

> if this needs to stay, can we cache these? no need to recreate the same image a bunch of times, and this kind of stuff can be expensive.

Not sure why we need to cache these. If an object needs to generate a lot of images form colors at some point then that would make sense.  Seems like YAGNI / pre-optimization. 
> If we don't want to do a shared cache object, we could at least wrap something domain-specific in a dispatch once (much like the other color categories). e.g. + (UIImage*)wmf_languageSelectionBackgroundImage

Pre-Optimization. Colors are a shared pallet. This isn't a shared pallet of images. This is why we have viewDidLoad - to do something once for a specific view.